packages/utils/src/initialize-cw-test-env/index.ts

```diff
@@ -48,7 +48,7 @@ async function main() {
         addressLine1: a.addressLine1 ?? "",
         addressLine2: a.addressLine2 ?? "",
         city: a.city ?? "",
-        state: a.state ?? "",
+        state: (a.state as USState) ?? "",
```

Comment:
Looking as to why do we have `state` as string here leads us to the `Address` domain at API.

There we have `state: string`, but it seems we can easily convert it to `USState` and minimize runtime errors in the future.